### ISSUE TITLE
[JavaScript] regular expression match too greedy?

### ISSUE THREAD

**Issue by iccir** [sublimehq/Packages#235]
With the following code:

```
(y - 1) / ((x - 1) / -2)
(y - 1) / ((x - 1) /  2)
 y      / ((x - 1) / -2)
```

The first line incorrectly highlights `/ ((x - 1) /` as a `string.regexp.js`.  The second and third lines are highlighted as expected.

**Comment by wbond** [comment]
This should be fixed by dde4ba2aad051f5fb89f59a1f74e8981f08bdb5b

**Comment by iccir** [comment]
Thanks!
